mobile/src/reducers/organization/department.model.ts

```diff
@@ -26,4 +26,8 @@ export class Department {
     @dataMember()
     @required({ nullable: true })
     public isHeadDepartment: boolean;
+
+    public equals(obj: Department): boolean {
```

Comment:
equals means when all instance's properties are equal to obj's properties, not only id